docs/testing/releases/740.md

```diff
@@ -0,0 +1,86 @@
+6. Enable a block theme (for example TT2).
+7. Now go to **Appearance -> Editor** and add the `Mini Cart` block in the header. Save.
+8. Check that the `Mini Cart` is translated (check with an empty cart and with a filled cart). Please, be sure that you are using a language that has translations for the Mini Cart Block. My suggestion is to use the Deutsch language.
+
+### Fix Featured Product and Featured Category buttons misalignment in Twenty Twenty Two theme. ([6156](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6156))
+
+1. Activate a block theme (ie: Twenty Twenty-Two).
+2. Add Featured Product and/or Featured Category blocks to a post or page.
+3. Verify the button is centered by default.
+4. Verify you can change the alignment of the button using the _Justify items_ dropdown inherited from the _Buttons_ block.
+![imatge](https://user-images.githubusercontent.com/3616980/160625173-f9ad42ed-b769-42e3-9ad8-3f3abe60b61c.png)
+5. Test again with a classic theme (ie: Storefront).
+
+### Remove the ToggleButtonControl in favor of ToggleGroupControl. ([5967](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5967))
+
+1. Check out this branch and compile the changes
+2. Activate a **block theme**, like Twenty Twenty Two
+3. Create a new page, and add the following blocks:
+   - Active Filters
+   - Attribute Filter
+   - Price Filter
+   - Product Categories
+   - Reviews
+4. Ensure that options are using the new component and are working correctly
+
+## Feature Plugin
+
+### The order summary area for the Cart and Checkout Blocks is now powered by Inner Blocks allowing for more customizations and extensibility. ([6065](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6065))
+
+1. Go to a page and add the Checkout block.
+2. Go to a page and add the Cart block
+3. In the editor, inside each block, select the Order summary and notice that you can select the following inner blocks:
+    | Cart | Checkout |
+    | --------------- | --------------- |
+    |  Heading, Subtotal, Cart Items, Fees, Coupon Form, Shipping, Taxes | Subtotal, Cart Items, Fees, Coupon Form, Shipping, Taxes |
+4. Try to remove or move some of the inner blocks and save.
+5. Check on the website that your changes are reflected. Note that Taxes, Fees and Coupons will appear on the website only if the shop supports them.
+6. For the Cart > Shipping inner block make sure you can enable disable the shipping calculator and that the change is visibile in the website
+7. Enable Taxes on your website (`/wp-admin/admin.php?page=wc-settings`) by clicking  `Enable tax rates and calculations`
+8. Go to the C & C blocks and select the Taxes inner block. 
+9. In the inner block's setting toggle on / of `Show rate after tax name` and make sure that is reflected in the website
+10. Try to place an order and make sure it's successful.
```

Comment:
I noticed that the Translations are not loading in the inspector when selecting these inner blocks. @ralucaStan I am just wondering because `Subtotal` should definitely be translated already, but it's not. What do you think? Should we make an issue to investigate?
##### Checkout 
The testing notes say check I can remove Inner Blocks but I cannot remove any of them with Gutenberg active, I can only move them. I also cannot remove them even if I unlock them using Gutenberg. Without Gutenberg, not all blocks can be removed, only `Coupon Form` can be removed. 
##### Cart
Can't remove inner blocks beside coupon form either, cc @ralucaStan are the testing instructions fully correct?

Reply:
> I noticed that the Translations are not loading in the inspector when selecting these inner blocks. @ralucaStan I am just wondering because Subtotal should definitely be translated already, but it's not. What do you think? Should we make an issue to investigate?


Sure, handling this in a separate issue is the way to go.


> The testing notes say check I can remove Inner Blocks but I cannot remove any of them [...] cc @ralucaStan are the testing instructions fully correct


Well spotted @opr, the testing instructions are incorrect. I'll update. For Cart only Order Summary Heading & Coupon form can be removed, and for Checkout only the Coupon form.

@tarhi-saad I've updated the testing step 4 from [the PR ](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6065)

from

> 4. Try to remove or move some of the inner blocks and save.

to

> 4. Try to move some of the inner blocks and save. Confirm that in the Cart block only Order Summary Heading and the Coupon form inner blocks can be removed, and for the Checkout block only the Coupon form.

Reply:
Updated!